packages/utils/adapters/src/adapters.ts

```diff
@@ -1,10 +1,10 @@
 import { TextProps, ViewProps, ImageProps } from 'react-native';
 import { IFilterMask } from './filter.types';
-import type { ITextWin32Props } from '@office-iss/react-native-win32';
+import type { ITextWin32Props, IViewWin32Props } from '@office-iss/react-native-win32';
 
 // export core interface types
 export type ITextProps = TextProps & Partial<ITextWin32Props>;
-export type IViewProps = ViewProps;
+export type IViewProps = ViewProps & IViewWin32Props;
```

Comment:
Make this `Partial<IViewWin32Props>` like we did for Text? I don't think there's any required props in IViewWin32Props but it might be worth to safeguard this in case that changes.